daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/preprocessing/ValueTranslator.scala

```diff
@@ -210,7 +210,7 @@ private[engine] final class ValueTranslator(
                       mbLbl.foreach(lbl_ =>
                         if (lbl_ != lbl)
                           typeError(
-                            s"Mismatching record label $lbl_ (expecting $lbl) for record $tyCon"
+                            s"Mismatching record field label $lbl_ (expecting $lbl) for record $tyCon"
```

Comment:
Everywhere else it is "record field", but here "record field label" -- should we remove "label" from here?
Also add single quotes around `$lbl_` and `$lbl`

Reply:
quotes got lost during rebasing, added it.
As menitoned in the other comment, I don’t think `record field` here makes sense. We have a field in the right position it just has the wrong name. This is the case where we match fields by order. In the other instance, we match fields by name rather than order and there is no field with the given name so there we really don’t have a field.